tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -10,7 +10,7 @@ parameters:
 
 - name: testPackage
   type: string
-  default: "@fluidframework/test-end-to-end-tests"
+  default: "@fluid-internal/test-end-to-end-tests"
```

Comment:
Can you confirm that the pipelines don't require the package to be published on npm? Another approach would be to make these changes on a test branch and try running from there first

Reply:
I have merged my changes into a new branch in microsoft/FluidFramework and ran the pipelines for client-packages and Build - test-tools. both ran successfully